core/common/src/main/java/alluxio/uri/SingleMasterAuthority.java

```diff
@@ -66,11 +63,11 @@ public boolean equals(Object o) {
 
   @Override
   public int hashCode() {
-    return Objects.hashCode(mAuthority);
+    return Objects.hashCode(toString());
```

Comment:
Can do this more efficiently by passing `mHost`, `mPort`

Reply:
I wasn't sure if the hash code was used anywhere else and I wanted to keep the behavior the same as it was before in case it broke anything. I will change this if you think those parameters for the hashCode are better.

Reply:
good call - it would be bad to change if we were persisting the hash code anywhere. In this case I know for certain that we aren't, and the general convention in our codebase for implementing `hashCode` is to hash the fields compared in `.equals()`